Split/Localhost/LocalhostSplitsStorage.swift

```diff
@@ -13,6 +13,7 @@ class LocalhostSplitsStorage: SplitsStorage {
     var changeNumber: Int64 = -1
     var updateTimestamp: Int64 = 1
     var splitsFilterQueryString: String = ""
+    var flagsSpec: String = ""
```

Comment:
Should it be "1.1"? (or a reference to a string const "1.1" to reuse in other places too)

Reply:
This is for localhost only so I don't think it's relevant ?